_posts/2020-Q1-open-source-report.markdown

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: 2020-q1-open-source-report
+date: 2020-04-09 20:12:12
+categories: [open-source]
+author: bronzdoc
+
+---
+
```

Comment:
@bronzdoc I think this article needs an introduction. This is the first report that we do, so no one is familiar with it.

Reply:
@bronzdoc also, you're missing the `<!--more-->` tag. It shows the entire article in the index page without it. We don't want to have the image before that tag.